python-checks/src/main/java/org/sonar/python/checks/hotspots/SecureCookieCheck.java

```diff
@@ -104,14 +107,30 @@ public void initialize(Context context) {
   }
 
   @CheckForNull
-  private static RegularArgument getSecureArgument(List<Argument> arguments) {
+  private static RegularArgument getSecureArgument(List<Argument> arguments, int nArg) {
     return arguments.stream()
+      // in call site of this function, argument will always be of type RegularArgument because we check no argument is unpacking
       .map(RegularArgument.class::cast)
       .filter(argument -> {
         Name keywordArgument = argument.keywordArgument();
         return keywordArgument != null && keywordArgument.name().equals(SECURE);
       })
-      .findFirst().orElse(null);
+      .findFirst().orElse(checkSensitivePositionalArgument(arguments, nArg));
```

Comment:
Could you use `Optional#orElseGet` instead of `orElse` ? As the latter is always evaluated no matter whether the former is present or not.

Reply:
good catch! thanks!